static/css/master.css

```diff
@@ -1696,7 +1696,7 @@ fieldset.home legend {
 * html .ui-helper-clearfix { height:1%; }
 .ui-helper-clearfix { display:block; }
 
-.ui-helper-zfix { width: 100%; height: 100%; top: 0; left: 0; position: absolute; opacity: 0; filter:Alpha(Opacity=0); }
+.ui-helper-zfix { width: 100%; height: 100%; top: 0; left: 0; position: absolute; opacity: 0; filter: alpha(opacity=0); }
```

Comment:
QUESTION: I guess we still need an absolutely positioned item?

Reply:
I think this is for a jquery dialogue box

Reply:
I don't think this selector is used anywhere, rm'ing